src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingOptions.cs

```diff
@@ -9,6 +9,11 @@ namespace Microsoft.AspNetCore.ResponseCaching
     public class ResponseCachingOptions
     {
         /// <summary>
+        /// The size limit for the response cache middleware in bytes. The default is set to 100 MB.
+        /// </summary>
+        public long SizeLimit { get; set; } = 100 * 1024 * 1024;
```

Comment:
Should we have ResponseCachingOptions extend MemoryCacheOptions so other options can be set?